dev/README.md

```diff
@@ -224,10 +224,8 @@ To build and run the infra setup command do:
 ```
 dev/import-images-tests-k3d
 # ./dev/create-zot-certs 'FleetCI-RootCA'
-pushd e2e/testenv/infra
-  go build
-popd
-./e2e/testenv/infra/infra setup
+dev/create-secrets
+go run ./e2e/testenv/infra/main.go setup
```

Comment:
Nice, this could be simplified in `.github/workflows/e2e{-nightly,}-ci.yml` and `dev/setup-single-cluster` as well.